scripts/package

```diff
@@ -4,8 +4,16 @@ set -e
 source $(dirname $0)/version
 cd $(dirname $0)/..
 
+ARCH=${ARCH:-"amd64"}
+SUFFIX="-${ARCH}"
```

Comment:
We already have `ARCH` and `SUFFIX` defined in the `version` script. We can get rid of these two lines.